fix(NewCollectiveModal): Mention that own user is part of collective

mejo- opened the pull request:
People got confused by searching their own user.

#### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
![screenshot](https://github.com/nextcloud/collectives/assets/3582805/2a8db5c2-4908-4bd2-8527-2f836502426f) | ![recording1](https://github.com/nextcloud/collectives/assets/3582805/f08cb865-17d8-4bf9-8c86-e6bf062ac6f5)


### 🏁 Checklist

- [x] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [x] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits

mejo- commented on src/components/Nav/NewCollectiveModal.vue:
```diff
@@ -87,6 +87,7 @@
 					<MemberPicker :show-selection="true"
 						:selected-members="selectedMembers"
 						:on-click-searched="onClickSearched"
+						:empty-content-hint="t('collectives', 'Your user is added automatically.')"
```
Not sure about the wording. Maybe "As creator, you're already added." is better? :thinking:

jancborchardt reviewed (commented):
Instead of as a message, we could show the owner's account directly below in the list like added accounts do? That would make it more obvious than a line of text.

Slightly related: We want to move from the wording "user" to either "account" or "person", could be replaced in the search field.

mejo- commented:
Thanks @jancborchardt, much better. Implemented now, see the screencast above. I also changed "users" to "accounts" where appropriate.

mejo- commented:
I wonder whether it would further help to change the "Create" button to something like "Create without adding people" in case no other members were selected.

juliushaertl commented:
Small addition, we could preserve the previously entered collective name in the dialog title to make it more obvious what this is about, e.g. "Add people to fox" instead of just "Add people".

Overall i find the different usages of account/people/members a bit confusing. 

<img width="704" alt="Screenshot 2023-11-09 at 19 56 28" src="https://github.com/nextcloud/collectives/assets/3404133/b3dcecdf-8df9-443a-b049-18e35fe6f179">


<img width="401" alt="Screenshot 2023-11-09 at 19 55 35" src="https://github.com/nextcloud/collectives/assets/3404133/b5fa4c25-f5b7-4d0a-b1c1-2fafe7e02dcf">

<img width="429" alt="Screenshot 2023-11-09 at 19 55 54" src="https://github.com/nextcloud/collectives/assets/3404133/065c0984-8c72-493c-a3ce-70b9f739775e">

Maybe we can somehow unify that? Pick one high level word (members) and a fixed set for the actual membership entities (account, group, circle)?

mejo- commented:
> Overall i find the different usages of account/people/members a bit confusing.
> Maybe we can somehow unify that? Pick one high level word (members) and a fixed set for the actual membership entities (account, group, circle)?


I agree, thanks for bringing this up. Your suggestion makes sense: account, group, circle for the different entities and "members" for all of them. So I'd replace "Add people" with "Add members" and search for more occurrences of "users" to be replaced by "accounts". Agreed?

mejo- commented:
> Small addition, we could preserve the previously entered collective name in the dialog title to make it more obvious what this is about, e.g. "Add people to fox" instead of just "Add people".

Great idea. Implemented now (using "Add members to <name>"). See updated screencast above.

> I agree, thanks for bringing this up. Your suggestion makes sense: account, group, circle for the different entities and "members" for all of them. So I'd replace "Add people" with "Add members" and search for more occurrences of "users" to be replaced by "accounts". Agreed?

Also implemented. See updated screencast above.
